Library/Homebrew/test/livecheck/strategy/xml_spec.rb

```diff
@@ -142,11 +183,11 @@
 
       # Returning an array of strings from block
       expect(xml.versions_from_content(content_version_text, regex) do |xml, regex|
-        xml.get_elements("versions//version").map { |item| item.text[regex, 1] }
+        xml.get_elements("/versions/version").map { |item| item.text[regex, 1] }
       end).to eq(content_matches)
 
       expect(xml.versions_from_content(content_version_attr, regex) do |xml, regex|
-        xml.get_elements("items//item").map do |item|
+        xml.get_elements("/items/item").map do |item|
```

Comment:
I assume these changes were made just because we're in the same file but aren't directly related to the new method that was added.

Reply:
That's correct. I added new tests for `Xml#element_text` that use `#get_elements`, so I figured I would align the format of the existing XPaths with the newly-added ones. Due to how the example XML content is structured, the modified XPaths achieve the same goal and the changes are just syntactical (i.e., taking an explicit rather than loose approach).